modules/ppcp-save-payment-methods/src/SavePaymentMethodsModule.php

```diff
@@ -340,11 +341,19 @@ function () use ( $c ) {
 
 				add_action(
 					'woocommerce_add_payment_method_form_bottom',
-					function () {
+					function () use ( $c ) {
 						if ( ! is_user_logged_in() || ! is_add_payment_method_page() ) {
 							return;
 						}
 
+						// Do not display PayPal button if the user already has a PayPal payment token.
```

Comment:
I would suggest to remove the inline comment and add it for the whole action to describe why this action is added.

```php
/**
 * Disables the PayPal button if the user already has a PayPal payment token.
 */
add_action(
					'woocommerce_add_payment_method_form_bottom',
					...
```